tests/Feature/RateLimiterTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Http\Integrations\GithubConnector;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Cache;
+use Saloon\Config;
+use Saloon\Enums\Method;
+use Saloon\Http\Faking\MockClient;
+use Saloon\Http\Faking\MockResponse;
+use Saloon\Http\Request;
+use Saloon\RateLimitPlugin\Exceptions\RateLimitReachedException;
+use Saloon\RateLimitPlugin\Limit;
+use Tests\TestCase;
+
+class GithubConnectorTestVersion extends GithubConnector
+{
+    protected function resolveLimits(): array
+    {
+        return [
+            Limit::allow(0)->everyHour(),
+        ];
+    }
+}
```

Comment:
For me its not good way, lets move limit into config and change config value in tests and test them.

Reply:
moved limit to config and changing it for tests